src/lib/SnackBarProvider/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { FC, PropsWithChildren, useCallback } from 'react';
+import { Snackbar } from 'react-native-paper';
+import { useSnackBarState } from 'lib/SnackBarProvider/recoil';
+import Typography from 'components/Typography';
+import { useTranslation } from 'react-i18next';
+import useStyles from './useStyles';
+
+export interface SnackBarProviderProps {}
+
+const SnackBarProvider: FC<PropsWithChildren<SnackBarProviderProps>> = ({ children }) => {
+  const { visible, onDismiss, text, duration } = useSnackBarState();
+  const { t } = useTranslation();
+  const styles = useStyles();
+
+  const onDismissAction = useCallback(() => {
+    if (onDismiss) {
+      onDismiss();
+    }
+  }, [onDismiss]);
+
+  return (
+    <>
+      {children}
+      <Snackbar
```

Comment:
Somehow the `hide` button is light-blue on iOS, can you align it with the theme colors?